checkov/terraform/runner.py

```diff
@@ -72,15 +73,15 @@ def __init__(
         self.external_registries = [] if external_registries is None else external_registries
         self.graph_class = graph_class
         self.parser = parser or TFParser()
-        self.definitions: dict[TFDefinitionKey, dict[str, Any]] | None = None
-        self.context: dict[TFDefinitionKey, dict[str, Any]] | None = None
+        self.definitions: dict[TFDefinitionKey, dict[str, Any]] | None = None  # type:ignore[assignment]  # need to check, how to support subclass differences
```

Comment:
maybe will need a generic usage to solve, like `GraphManager`?

Reply:
yes, correct. I just didn't want to open this pandora box yet 😅

Reply:
implemented it and it was not such a big change as I expected , lucky 😄